src/connection/utils.ts

```diff
@@ -68,6 +72,8 @@ export function getConnectionName(
       return 'Coinbase Wallet'
     case ConnectionType.WALLET_CONNECT:
       return 'WalletConnect'
+    case ConnectionType.WALLET_CONNECT_V2:
+      return 'WalletConnect V2'
```

Comment:
Isn't this displayed to the user, who should not care which version they are using?

```suggestion
    case ConnectionType.WALLET_CONNECT:
    case ConnectionType.WALLET_CONNECT_V2:
      return 'WalletConnect'
```

Reply:
Since they're both running side by side right now (not behind a flag), I wanted to differentiate their name so it's explicit which one is which within the modal. 

We're going to update this on Monday to not run side by side, and then, I will follow your suggestion.